reference/docs-conceptual/install/PowerShell-in-Docker.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using  PowerShell in Docker
+description: How to use PowerShell that is preinstalled in a Docker image.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 03/03/2020
+---
+
+# Using PowerShell in Docker
+
+We are publishing Docker images with PowerShell preinstalled. This article shows you how to get
+started using PowerShell in the Docker container.
+
+## Finding available images
+
+The released images require Docker 17.05 or newer. It is also expected that you are able to run
+Docker without `sudo` or local administrative rights. Please follow Docker's official
+[instructions][install] to install `docker` correctly.
+
+The release containers derive from the official distribution image, such as `centos:7`, then install
+dependencies, and finally install the PowerShell package.
+
+These containers live at [hub.docker.com/r/microsoft/powershell][docker-release].
+
+For more information about these Docker images, visit the [PowerShell-Docker][PowerShell-Docker]
+repository on GitHub.
+
+## Using PowerShell in a container
+
+The following steps show the Docker commands required to download the image and start an interactive
+PowerShell session.
+
+```console
+docker run -it mcr.microsoft.com/powershell
+```
+
+### Remove the image when no longer needed
+
+The following command is used to delete the Docker container when you no longer need it.
+
+```console
+docker rmi mcr.microsoft.com/powershell
+```
+
+## Legal and Licensing
+
+PowerShell is licensed under the [MIT license][].
+
+### Windows Docker File and Image Licenses
+
+By requesting and using the Container OS Image for Windows containers, you acknowledge,
+understand, and consent to the Supplemental License Terms available on Docker hub:
+
+- [Window Server Core](https://hub.docker.com/r/microsoft/windowsservercore/)
```

Comment:
Why not maintain consistency and use the other localized link system as all the other links on the page?